orchestrator/graphql/resolvers/subscription.py

```diff
@@ -58,11 +58,12 @@ def get_subscription_details(subscription: SubscriptionTable) -> SubscriptionInt
 
     subscription_details = SubscriptionModel.from_subscription(subscription.subscription_id)
     base_model = subscription_details.__base_type__ if subscription_details.__base_type__ else subscription_details
-    subscription_details = base_model.from_other_lifecycle(  # type: ignore
+    base_subscription_details = base_model.from_other_lifecycle(  # type: ignore
         subscription_details, SubscriptionLifecycle.INITIAL, skip_validation=True
```

Comment:
Why do we convert subscriptions to the initial lifecycle again?

Reply:
because it wants the nested models to be the exact instances.

`Expected value of type 'Sn8ServiceAttachPointBlockInactive' but got: <Sn8ServiceAttachPointBlock instance>.`